gradle.properties

```diff
@@ -1,4 +1,4 @@
-junitVersion=5.9.2
+junitVersion=5.10.1
```

Comment:
We do such upgrades in seperated PRs. As #799 is already merged, please rebase your branch - thanks you :)